src/tls13.c

```diff
@@ -4233,6 +4431,226 @@ static int Dtls13DoDowngrade(WOLFSSL* ssl)
 }
 #endif /* WOLFSSL_DTLS13 && !WOLFSSL_NO_CLIENT*/
 
+/* check if the server accepted ech or not */
+#if defined(HAVE_ECH)
+static int EchCheckAcceptance(WOLFSSL* ssl, const byte* input,
+  int serverRandomOffset, int helloSz)
+{
+    int ret = 0;
+    int digestType;
+    int digestSize;
+    HS_Hashes* tmpHashes;
+    HS_Hashes* acceptHashes;
+    byte zeros[WC_MAX_DIGEST_SIZE] = {0};
+    byte transcriptEchConf[WC_MAX_DIGEST_SIZE];
+    byte expandLabelPrk[WC_MAX_DIGEST_SIZE];
+    byte acceptConfirmation[8];
```

Comment:
8? Please comment on why or use enum / macro

Reply:
It is used below a few times

Reply:
accept confirmation is an 8 byte parameter in the ech spec, I'll add a define for it